types/react-native/index.d.ts

```diff
@@ -234,7 +234,7 @@ interface EventEmitter extends EventEmitterListener {
      *
      *   emitter.emit('someEvent', 'abc'); // logs 'abc'
      */
-    emit(eventType: string): void
+    emit(eventType: string,...param:any[]): void
```

Comment:
Fix the linter failure like so: `emit(eventType: string, ...param: any[]): void`